bin/ch-grow.py.in

```diff
@@ -465,6 +467,15 @@ class CLI_Print_Storage(ch.CLI_Action_Exit):
       print(namespace.storage)
       sys.exit(0)
 
+class CLI_List_Images(ch.CLI_Action_Exit):
+
+   def __call__(self, _, namespace, *args, **kwargs):
+      print("IMAGE") 
```

Comment:
Suggest we omit the column header since there is only one.

Reply:
That's fair. I had put it up with the intention of adding others.